libs/application/core/src/validation/validators.ts

```diff
@@ -20,7 +20,7 @@ function populateError(
   formatMessage: FormatMessage,
 ) {
   let errorObject = {}
-  error.forEach((element) => {
+  error?.forEach((element) => {
```

Comment:
Why null check here? this is strongly typed as an array of zod issues

Reply:
Remove it, forgot to take it out after testing